jobs/JREGIONAL_PROCESS_BUFR

```diff
@@ -62,8 +62,14 @@ the specified cycle.
 #
 if [ ${CYCLE_TYPE} == "spinup" ]; then
   workdir=${CYCLE_DIR}/process_bufr_spinup
+  if [ ${DO_ENSEMBLE} == "TRUE" ]; then
+     workdir=${CYCLE_DIR}/process_bufr_spinup_enkf
+  fi
 else
   workdir=${CYCLE_DIR}/process_bufr
+  if [ ${DO_ENSEMBLE} == "TRUE" ]; then
+     workdir=${CYCLE_DIR}/process_bufr_enkf
+  fi
```

Comment:
why the ensemble needs a separate process_bufr directory?

Reply:
control and ensemble both have process_bufr and they overwrite each other if use the same name.